src/standalone/plugins/top-bar/components/DarkMode.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * @prettier
+ */
+import React, { useState, useCallback } from "react"
+import LightBulb from "../assets/lightbulb.svg"
+import LightBulbOff from "../assets/lightbulb-off.svg"
+
+const LightBulbIcon = () => {
+  const [isDarkMode, setIsDarkMode] = useState(false)
```

Comment:
(Untested) set automatically [based on OS dark-mode](https://stackoverflow.com/questions/56393880/how-do-i-detect-dark-mode-using-javascript):

```suggestion
  const [isDarkMode, setIsDarkMode] = useState(() => window.matchMedia('(prefers-color-scheme: dark)').matches)
```